config/routes.rb

```diff
@@ -8,6 +8,9 @@
       resources :courses
       resources :subjects
       resources :lessons
+      resources :dictionaries
+      resources :words
+      resources :questions
```

Comment:
we don't main questions for admin. Admin has dashboards for different types of questions